.github/workflows/ci.yml

```diff
@@ -14,20 +14,14 @@ jobs:
     strategy:
       matrix:
         node-version:
-          - 7
-          - 8
-          - 9
-          - 10
-          - 11
-          - 13
-          - 14
-          - 15
+          - 20
```

Comment:
Think it might be worth include 22, since that's current LTS now?

Reply:
Will do this manually